app/main.py

```diff
@@ -1,6 +1,33 @@
+import os
+from dotenv import load_dotenv
+import requests
+
+load_dotenv()
+
+URL = "https://api.weatherapi.com/v1/current.json"
+API_KEY = os.getenv("API_KEY")
+CITY = "Paris"
+PARAMS = {
+    "key": API_KEY,
+    "q": CITY,
+}
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    request = requests.get(url=URL, params=PARAMS).json()
+
+    city = request.get("location")["name"]
+    country = request.get("location")["country"]
```

Comment:
you can create separate variables for location and current and check if they are  not None before accessing their keys